swagger/swagger.yaml

```diff
@@ -27,8 +27,136 @@ paths:
+          responses:
+            '200':
+              description: Show one property
+              content:
+                application/json:    
+                  schema:
+                    $ref: "#/components/schemas/ResponseProperties"
+            default:
+              description: unexpected error
+              content:
+                application/json:
+                  schema:
+                    $ref: "#/components/schemas/Error"  
+      patch:
+          summary: Updating specific property
+          operationId: updateProperty
+          parameters:
+          - name: PropertyId
+            in: path
+            description: property ID
+            required: true
+            schema:
+              type: string
+              format: int64
+          tags:
+          - properties
+          requestBody:
+            description: the property to update
+            required: true
+            content:
+              application/json:    
+                schema:
+                  $ref: "#/components/schemas/listings"
+          responses:
+            '201':
+              description: req.param upserted
+              content:
+                application/json:    
+                  schema:
+                    $ref: "#/components/schemas/ResponseListings"
+            '200':
+              description: OK
+              content:
+                application/json:    
+                  schema:
+                    $ref: "#/components/schemas/ResponseListings" 
+            '204':
+              description: No content
+              content:
+                application/json:    
+                  schema:
+                    $ref: "#/components/schemas/ResponseListings"              
+            default:
+              description: unexpected error
+              content:
+                application/json:
+                  schema:
+                    $ref: "#/components/schemas/Error"
+    delete:
+          summary: Delete specific property
+          operationId: deleteProperty
+          parameters:
+          - name: propertyId
+            in: path
+            description: Property ID
+            required: true
+            schema:
+              type: string
+              format: int64 
+          tags:
+          - properties
+          responses:
+            '200':
+              description: deleted
+              content:
+                application/json:    
+                  schema:
+                    $ref: "#/components/schemas/ResponseListings"
+            '204':
+                description: Cant find id
+                content:
+                  application/json:    
+                    schema:
+                      $ref: "#/components/schemas/Error"                 
+            default:
+              description: unexpected error
+              content:
+                application/json:
+                  schema:
+                    $ref: "#/components/schemas/Error"
```

Comment:
Saknar ju schemat för listingen nedan:

lägg in alla properties för listingen under schemas. Tänk på att Coordinate är av typen object som innehåller properties latitude och longitude